aries-site/src/pages/design-tokens/versioning.mdx

```diff
@@ -25,17 +25,38 @@ import { Example } from '../../layouts';
 
 ## Production-ready versions
 
+### ^2.0.0
+
+This release, referred to as "v2-Landmark," provides production-ready support for HPE's refreshed visual identity (October 2025 - present). It introduces the new Landmark color palette and the HPE Graphik font face to align with the updated brand expression.
+
+For Grommet applications, v2-Landmark is available by [grommet-theme-hpe v8.x](https://github.com/grommet/grommet-theme-hpe/releases/tag/v8.0.0)
+
+In the [HPE Design System Figma Component Library](https://www.figma.com/design/HDckqS2MWhINfC8EIQPMV1/HPE-Design-System-Components-V2?m=auto&t=NjHwxPFWYTvhZTP5-6), this is the default theme.
+
 ### ^1.0.0
 
-This release provides production-ready support with styles in alignment with the latest HPE theme (February 2025 - present).
+<Notification
+  status="info"
+  message="This HPE theme was introduced in February 2025. Teams should coordinate with their product management and HPE brand for upgrade timelines to v2-Landmark."
+  actions={[
+    { label: 'All design tokens', href: '/design-tokens/all-design-tokens' },
+  ]}
+  width={{ max: 'xlarge' }}
+/>
 
-In the [new HPE Design System Figma Component Library](https://www.figma.com/design/HDckqS2MWhINfC8EIQPMV1/HPE-Design-System-Components-V2?m=auto&t=NjHwxPFWYTvhZTP5-6), this is the default theme.
+This HPE theme was introduced in February 2025.
```

Comment:
Can we try to add whitespace between the notification and the following sentence? Not sure why its not including it now, but you may be able to add a `<br />` to address.

<img width="840" height="363" alt="Screenshot 2025-11-05 at 8 41 35 AM" src="https://github.com/user-attachments/assets/c910e2c1-2842-4cc2-ad74-2077cdd20d6b" />